src/main/java/com/develhope/spring/vehicle/models/TractionControl.java

```diff
@@ -0,0 +1,26 @@
+package com.develhope.spring.vehicle.models;
+
+import com.develhope.spring.model.vehicle.optionals.Optionals;
+
+public class TractionControl implements Optionals {
+    private boolean enabled;
+
+    public TractionControl(boolean enabled) {

```

Comment:
Il costruttore potrebbe avere "enabled =  true" visto che crei l'oggetto soltanto se vuoi inserirlo come optional. In questo modo non devi passare nulla a parametro e col setter puoi sempre cambiarlo qualora fosse necessario. 
Questa è semplicemente una proposta, non un cambio obbligatorio.